### PR TITLE
[SCIP] format to trigger build

### PR DESCRIPTION
**Pull request by matbesancon**
Trying to understand the error when merging the last SCIP PR

### CODE REVIEW

File: S/SCIP/build_tarballs.jl [hunk 1]
```diff
@@ -7,19 +7,35 @@ version = v"800.0.401"
...
+
 if [[ "${target}" == *86*-linux-gnu ]]; then
-   export LDFLAGS="-lrt"
-elif [[ "${target}" == *-mingw* ]]; then
-   # this is required to link to bliss on mingw
-   export LDFLAGS=-L${libdir}
+    if [[ "${nbits}" == 32 ]]; then
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib"
+    else
+        export CFLAGS="-Wl,-rpath-link,/opt/${target}/${target}/lib64"
```
Comment (fingolfin) [inline S/SCIP/build_tarballs.jl:37]: I think `-rpath-link` may be a false lead here. The error says:
```
... warning: libdl.so.2, needed by /workspace/destdir/lib/libipopt.so, not found ...
```
So my first question would be: is `libdl.so.2` actually around anywhere?  If not then the use `-rpath-link` is moot anyway.

**Comment by matbesancon** [comment]
Good catch thanks a lot!

**Comment by giordano** [comment]
I think there was something very wrong until https://github.com/JuliaPackaging/BinaryBuilderBase.jl/commit/fec32a342ef43c6c16d8564bf2cde089236fced7: with
```diff
diff --git a/src/Runner.jl b/src/Runner.jl
index 4e9169a..1959292 100644
--- a/src/Runner.jl
+++ b/src/Runner.jl
@@ -524,6 +524,7 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
         # Yes, it does seem that the inclusion of `/lib64` on `powerpc64le` was fixed
         # in GCC 6, broken again in GCC 7, and then fixed again for GCC 8 and 9
         gcc_version, llvm_version = select_compiler_versions(p, compilers)
+        @show gcc_version
         if arch(p) == "powerpc64le" && Sys.islinux(p) && gcc_version.major in (4, 5, 7)
             append!(flags, String[
                 "-L/opt/$(aatriplet(p))/$(aatriplet(p))/sys-root/lib64",
```
I get
```console
% julia --compile=min -e 'using BinaryBuilderBase; BinaryBuilderBase.runshell(Platform("powerpc64le", "linux"); preferred_gcc_version = v"6")'
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
gcc_version = v"5.2.0"
sandbox:${WORKSPACE} # gcc --version
powerpc64le-linux-gnu-gcc (GCC) 6.1.0
Copyright (C) 2016 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```
which explains why the flag was injected also for GCC 6.

If anything, with https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/324 we now get gcc_version = v"6.1.0" as expected.

**Comment by giordano** [comment]
Proper fix at https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/333

File: S/SCIP/build_tarballs.jl [hunk 2]
```diff
@@ -20,6 +23,9 @@ if [[ "${target}" == *86*-linux-gnu ]]; then
 elif [[ "${target}" == *-mingw* ]]; then
    # this is required to link to bliss on mingw
    export LDFLAGS=-L${libdir}
+elif [[ "${target}" == powerpc64le-linux* ]]; then
+   # needed to avoid dladdr@GLIBC_2.17 errors
+   export LDFLAGS="-L/opt/${target}/${target}/sys-root/lib64 -Wl,-rpath-link,/opt/${target}/${target}/sys-root/lib64"
```
Comment (giordano) [inline S/SCIP/build_tarballs.jl:28]:
```suggestion
```

**Comment by matbesancon** [comment]
thanks both
sadly, bad luck so far on the windows issue that @benlorenz  had spotted, we still seem to run into the infinite loop

**Pull request merged**